src/app.css

```
@@ -133,4 +133,5 @@ body {
 }
 
 /* Stream Video React SDK styles */
-@import '@stream-io/video-react-sdk/dist/css/styles.css';
\ No newline at end of file
+@import '@stream-io/video-react-sdk/dist/css/styles.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)